.gitignore

```diff
@@ -1,33 +1,4 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
-# dependencies
```

Comment:
Why have you removed all of these? I think some of it are needed for next.

Reply:
I haven't :/ It was serverless script